feature-pack/src/main/resources/modules/system/layers/base/io/smallrye/fault-tolerance/main/module.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2019, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<module xmlns="urn:jboss:module:1.9" name="io.smallrye.fault-tolerance">
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+    <resources>
+        <artifact name="${io.smallrye:smallrye-fault-tolerance}"/>
+    </resources>
+    <dependencies>
+        <module name="com.netflix.hystrix.core"/>
+        <module name="io.reactivex.rxjava"/>
+        <module name="io.smallrye.config"/>
+        <module name="io.smallrye.metrics"/>
```

Comment:
Further to this I don't see io.reactivex.rxjava in https://github.com/smallrye/smallrye-fault-tolerance/blob/2.1.3/pom.xml

Not needing two variants of RxJava in WildFly would be better.

Reply:
I missed that hystrix has the rxjava dep. I still don't see why it's needed here and it would be nice to be rid of it altogether in favor of rxjava2, but I can imagine getting rid of it altogether might be a problem.

Reply:
@bstansberry Dependencies in maven can have other dependencies – its a dependency brought in by Hystrix. SR just didn't make it explicit which they should since they use it too (in `CompositeObservableCommand` and `HystrixCommandInterceptor`).

```
[INFO] io.smallrye:smallrye-fault-tolerance:jar:2.1.4-SNAPSHOT
[INFO] +- org.eclipse.microprofile.fault-tolerance:microprofile-fault-tolerance-api:jar:2.0.2:compile
[INFO] +- com.netflix.hystrix:hystrix-core:jar:1.5.12:compile
[INFO] |  +- org.slf4j:slf4j-api:jar:1.7.0:compile
[INFO] |  +- com.netflix.archaius:archaius-core:jar:0.4.1:compile
[INFO] |  |  \- commons-configuration:commons-configuration:jar:1.8:compile
[INFO] |  |     +- commons-lang:commons-lang:jar:2.6:compile
[INFO] |  |     \- commons-logging:commons-logging:jar:1.1.1:compile
[INFO] |  +- io.reactivex:rxjava:jar:1.2.0:compile
[INFO] |  \- org.hdrhistogram:HdrHistogram:jar:2.1.9:compile
```

Reply:
@bstansberry Its needed by the classes mentioned above^. This will be get ridden off with SR FT .next soonish.

Reply:
@pferraro They are necessary until SR FT is rewritten. Removing that dependency fails with:
```
    Caused by: org.jboss.weld.exceptions.DeploymentException: WELD-001408: Unsatisfied dependencies for type MetricRegistry with qualifiers @Default
  at injection point [UnbackedAnnotatedField] @Inject io.smallrye.faulttolerance.metrics.MetricsCollectorFactory.registry
  at io.smallrye.faulttolerance.metrics.MetricsCollectorFactory.registry(MetricsCollectorFactory.java:0)
```